tests/test_widgets.py

```diff
@@ -101,6 +101,10 @@ def testNumberValidation(self):
         self.assertFalse(self.height.valid)
         self.assertRaises(ValueError, lambda: self.height.value)
 
+    def testToolTip(self):
+        self.assertEqual(self.height.form_lineedit.toolTip(), dummy)
+        self.assertIsNotNone(self.weight.form_lineedit.toolTip())
```

Comment:
It is better to check that the tooltip is empty `self.assertEqual(self.weight.form_lineedit.toolTip(), '')` which also ensures it is not None